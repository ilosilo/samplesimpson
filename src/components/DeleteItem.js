import * as React from 'react';

import { deleteItem } from '../config/Storage'
import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
import {
    View,
} from 'react-native';

const DeleteItem = (props) => {

    

    return (
        <View>
            <Icon name='delete' size={30} color='white' />
        </View>
    );
};

export default DeleteItem;
