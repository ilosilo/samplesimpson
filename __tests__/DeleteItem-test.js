
 import 'react-native';
 import React from 'react';
 import DeleteItem from '../src/components/DeleteItem';
 import renderer from 'react-test-renderer';
 
 test('renders correctly', () => {
    const tree = renderer.create(<DeleteItem />).toJSON();
    expect(tree).toMatchSnapshot();
  });