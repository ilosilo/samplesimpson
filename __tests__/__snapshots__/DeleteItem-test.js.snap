// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": "white",
          "fontSize": 30,
        },
        undefined,
        Object {
          "fontFamily": "Material Design Icons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ó°†´
  </Text>
</View>
`;
